validator/Cargo.toml

```diff
@@ -16,3 +16,7 @@ idna = "0.1"
 serde = "1"
 serde_derive = "1"
 serde_json = "1"
+card-validate = { version = "1.0", optional = true }
+
+[features]
+credit_cards = [ "card-validate" ]
```

Comment:
looking at `card-validate` dependencies, I think it's fine to depend on it directly without the features, it only adds one dep